hawkbit-repository/hawkbit-repository-jpa/src/test/java/org/eclipse/hawkbit/repository/jpa/ControllerManagementTest.java

```diff
@@ -536,6 +551,28 @@ public void findOrRegisterTargetIfItDoesNotExistThrowsExceptionForInvalidControl
                 .as("register target with too long controllerId should fail");
     }
 
+    @Test
+    @Description("Tries to register a target with an invalid controller id")
+    public void findOrRegisterTargetIfItDoesNotExistWithNameThrowsExceptionForInvalidControllerIdParam() {
```

Comment:
remove this test, because it does not make sense due to `findOrRegisterTargetIfItDoesNotExist` overloading

Reply:
Removed.